_includes/code/howto/pq-compression.py

```diff
@@ -0,0 +1,144 @@
+# THIS FILE NEEDS TESTS
+
+# ================================
+# ===== INSTANTIATION-COMMON =====
+# ================================
+
+# ===== Instantiation shown on snippet
+import weaviate
+import json, os
+
+# Instantiate the client with the OpenAI API key
+client = weaviate.connect_to_local(
+    port=8080,
+    grpc_port=50051,
+    headers={
+        "X-OpenAI-Api-Key": os.environ[ "OPENAI_API_KEY" ] # Replace with your inference API key
+    }
+)
+
+# ==============================
+# =====  DOWNLOAD DATA =====
+# ==============================
+
+# START DownloadData
+
+import requests
+import json
+
+# Download the data
+resp = requests.get( 'https://raw.githubusercontent.com/weaviate-tutorials/intro-workshop/main/data/jeopardy_1k.json' )
+
+# Load the data so you can see what it is
+data = json.loads(resp.text)
+
+# Parse the JSON and preview it
+print( type( data ), len( data ) )
+print( json.dumps( data[1], indent=2 ) )
+
+# END DownloadData
+
+# ==============================
+# =====  CONNECT =====
+# ==============================
+
+# START ConnectCode
+import weaviate, os, json
+import weaviate.classes as wvc
+
+client = weaviate.connect_to_local(
+    headers={
+        "X-OpenAI-Api-Key": os.environ['OPENAI_API_KEY'] # Replace with your OpenAI API key
+    }
+)
+
+client.is_ready()
+
+# END ConnectCode
+
+# ==============================
+# =====  INITIAL SCHEMA =====
+# ==============================
+
+# START InitialSchema 
+
+client.collections.create(
+    name="JeopardyCategory",
```

Comment:
I think the collection name should be `"Question"` for consistency with the V3 client examples. 

The references below to `category` and subsequent names should also change.